packages/emptystate/src/react/__specs__/storyshots.spec.js

```diff
@@ -6,8 +6,6 @@ import initStoryshots, {
 jest.mock('@pluralsight/ps-design-system-storybook-addon-center')
 jest.mock('@pluralsight/ps-design-system-storybook-addon-theme')
 
-jest.mock('../resize-observer')
```

Comment:
why the mock removal for storyshots?

Reply:
before it was an hoc and therefore muddied up my snapshots with elements. now that it's a hook the snapshot is simplier